lib/events/identify.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:logging/logging.dart';
+
+class IdentifyOperation {
+  final String operationType;
+
+  const IdentifyOperation._(this.operationType);
+
+  static const IdentifyOperation set = IdentifyOperation._("\$set");
+  static const IdentifyOperation setOnce = IdentifyOperation._("\$setOnce");
+  static const IdentifyOperation add = IdentifyOperation._("\$add");
+  static const IdentifyOperation append = IdentifyOperation._("\$append");
+  static const IdentifyOperation clearAll = IdentifyOperation._("\$clearAll");
+  static const IdentifyOperation prepend = IdentifyOperation._("\$prepend");
+  static const IdentifyOperation unset = IdentifyOperation._("\$unset");
+  static const IdentifyOperation preInsert = IdentifyOperation._("\$preInsert");
+  static const IdentifyOperation postInsert = IdentifyOperation._("\$postInsert");
+  static const IdentifyOperation remove = IdentifyOperation._("\$remove");
+}
+
+class Identify {
+  Set<String> propertySet = {};
+  Map<String, dynamic> properties = {};
+  final log = Logger('Amplitude-Identify');
```

Comment:
Not super familar with Dart, but should we be using the logger passed in by the Configuration instead?